repairnator/repairnator-realtime/src/main/java/fr/inria/spirals/repairnator/realtime/RTScanner.java

```diff
@@ -343,7 +362,15 @@ public void submitBuildToExecution(Build build) {
 
         if (failing) {
             LOGGER.info("Failing or erroring tests has been found in build (id: "+build.getId()+")");
-            this.pipelineRunner.submitBuild(build);
+            if (this.pipelineMode.equals(PIPELINE_MODE.DOCKER)) {
+                this.dockerPipelineRunner.submitBuild(build);
+            } else {
+                try {
+                    this.ActiveMQPipelineRunner.submitBuild(build);
+                } catch(Exception e){
+                    LOGGER.warn("Failed to send message to ActiveMQ queue");
```

Comment:
would prefer to throw in presence of exception, remove try/catch

Reply:
Hi, the exception need to be caught somewhere. I found out that this try catch block is actually unneccessary, since the JMSException is already caught inside ActiveMQPipelinerRunner submit().